compiler-core/src/docs.rs

```diff
@@ -164,6 +167,7 @@ pub fn generate_html(
             .ast
             .definitions
             .iter()
+            .filter(|statement| !statement.is_internal())
```

Comment:
nit: is_public() would be a wee bit clearer here

Reply:
Aha the problem is that we want to do it with both public and internals, so anything that is not private, how would you call such a method?
```rust
fn wibble(&self) -> bool {
  case self {
    Publicity::Internal | Publicity::Public => true,
    Publicity::Private => false,
  }
}
```

Reply:
I realised this for the later ones 😓 

I wonder if `!x.is_internal()` should be `x.is_importable()` or something?

Reply:
Ah I really like that name, I'll go for it and we can always change it later 😁